arcgis-ios-sdk-samples/Maps/Show location history/README.metadata.json

```diff
@@ -26,7 +26,9 @@
         "AGSPolylineBuilder",
         "AGSSimpleLineSymbol",
         "AGSSimpleMarkerSymbol",
-        "AGSSimpleRenderer"
+        "AGSSimpleRenderer",
+        "AGSSimulatedLocationDataSource"
+
```

Comment:
There's an extra blank line here.

```suggestion
```

Reply:
- arcgis-ios-sdk-samples/Layers/Display subtype feature layer/README.metadata.json

The changes on `v.next` was a typo fix, which is also fixed in the latest update on `philium/readme-updates`. Furthermore, the latest update used the script to standardize the format, so the latest changes are adapted instead of using the version from `v.next`.

- arcgis-ios-sdk-samples/Maps/Read a geopackage/README.md

Similar situation. The latest update added more information to the README, so it is used instead of from `v.next`.

Reply:
Thanks for the explanation, but I'm not quite sure that's right. For example, the name of the subtype feature layer sample is inconsistent in this branch:

- "Display subtype feature layer" (`ContentPList.plist`)
- "Display a subtype feature layer" (`README.md`, `README.metadata.json`)

The metadata file in `v.next` has "Display subtype feature layer", which I assume is the correct one.

Reply:
Hmm that is true. 
 
I believe it is due to the initial miscommunication when I made changes to the titles in README, and then it propagated into metadata by the automated script. 

The content in metadata on `philium/readme-updates`  should reflect the most up-to-date version. But the discrepancies between `content.plist` and README titles should be fixed. Let me write a script to unify the titles.